pkg/controller.v1beta2/tensorflow/pod.go

```diff
@@ -175,6 +181,19 @@ func (tc *TFController) createNewPod(tfjob *tfv1beta2.TFJob, rt, index string, s
 	}
 	setRestartPolicy(podTemplate, spec)
 
+	// if gang-scheduling is enabled:
+	// 1. if user has specified other scheduler, we report a warning without overriding any fields. 
+	// 2. if no SchedulerName is set for pods, then we set the SchedulerName to "kube-batch".
+	if tc.Config.EnableGangScheduling {
+		if otherSchedulerNameExists(tfjob) {
```

Comment:
better name? May be IsGangSchedulerSet ?

Reply:
Sure